src/Latte/Macros/CoreMacros.php

```diff
@@ -26,6 +26,7 @@
  * - {_expression} echo translation with escaping
  * - {attr ?} HTML element attributes
  * - {capture ?} ... {/capture} capture block to parameter
+ * - {compress} ... {/compress} compresses output
```

Comment:
„compresses output“ is confusing, what about „removes insignificant HTML whitespace“